ED/protocol.html

```diff
@@ -1465,6 +1465,11 @@ <h2 property="schema:name">Changelog</h2>
                     <td><a href="#relationship-to-ldp" rel="spec:changeSubject">#relationship-to-ldp</a></td>
                     <td property="spec:statement">Add section describing the <a href="#relationship-to-ldp">the relationship between Solid Protocol and LDP</a>.</td>
                   </tr>
+                  <tr about="#b0af4fa6-5d35-47b5-8fe3-ad597dc8bdb5" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-3" rel="spec:changeClass" resource="spec:ChangeClass3">3</a></td>
```

Comment:
I believe the current state of the PR falls under correction class 2 in that the change does not functionally affect interpretation.

```suggestion
                    <td><a href="https://www.w3.org/Consortium/Process/#class-2" rel="spec:changeClass" resource="spec:ChangeClass2">2</a></td>
```

But if there is doubt, we can change it back to class 3.